scripts/kevm-integration.sh

```diff
@@ -17,14 +17,10 @@ git clone --recurse-submodules 'https://github.com/kframework/evm-semantics' $EV
 
 cd $EVM_SEMANTICS
 
-(   cd .build/k
-    (   cd haskell-backend/src/main/native/haskell-backend
-        git fetch $TOP
-        git checkout FETCH_HEAD
-    )
-    git add haskell-backend/src/main/native/haskell-backend
-    ci_git commit --message '!!! haskell-backend/src/main/native/haskell-backend: integration testing haskell backend'
-)
+# Use the K Nightly build from the Kore integration tests.
+rm -rf .build/k
+mkdir -p .build/k/k-distribution/target/release
+ln -s $TOP/.build/k .build/k/k-distribution/target/release
```

Comment:
We can see if it works, but we may want to instead `cp -r` the directory into place, because the scripts sometimes depend on things like `$0`, which may not resolve correctly.

Reply:
This is how I always set up my `evm-semantics` tree locally (`.build/k/k-distribution/target/release/k` is a symlink) but I can change it if you want.

Reply:
If it works locally for you, sounds good to me :+1: